eng/build.sh

```diff
@@ -230,6 +242,7 @@ function BuildSolution {
     /p:Publish=$publish \
     /p:UseRoslynAnalyzers=$enable_analyzers \
     /p:ContinuousIntegrationBuild=$ci \
+    /p:BootstrapBuildPath="$bootstrap_dir" \
```

Comment:
This isn't a property that Arcade honors.

Reply:
OK, have removed from `Build.ps1` too

Reply:
It's in `build-utils.ps1` - should I remove it from there too?

Reply:
Yes, please.  I think it made its way into build-utils when I copy/pasted from Roslyn.